tests/system/feature_store/test_spark_engine.py

```diff
@@ -1188,6 +1188,28 @@ def test_ingest_with_steps_drop_features(self):
         csv_path_storey = measurements.get_target_path(name="csv")
         read_and_assert(csv_path_spark, csv_path_storey)
 
+        measurements = fstore.FeatureSet(
+            "measurements_spark",
+            entities=[fstore.Entity(key)],
+            timestamp_key="timestamp",
+            engine="spark",
+        )
+        measurements.graph.to(DropFeatures(features=[key]))
+        source = ParquetSource("myparquet", path=self.get_remote_pq_source_path())
+        targets = [CSVTarget(name="csv", path=csv_path_spark)]
```

Comment:
i don't think you need to set targets explicitly for this test, can use the defaults.

Reply:
solved